download-build/action.yml

```diff
@@ -26,11 +30,21 @@ runs:
         JFROG_DL_BUILD: ${{ github.event.repository.name }}/${{ inputs.build-number }}
         JFROG_DL_EXCLUSIONS: ${{ inputs.exclusions }}
         JFROG_DL_REMOTE_REPO: ${{ inputs.remote-repo }}
-      run: jfrog rt download
-        --fail-no-op
-        --build "${JFROG_DL_BUILD}"
-        --exclusions "${JFROG_DL_EXCLUSIONS}"
-        "${JFROG_DL_REMOTE_REPO}/"
+      run: |
+        if [[ "${{ inputs.flat-download }}" == "true" ]]; then
+          jfrog rt download \
+            --fail-no-op \
+            --build "${JFROG_DL_BUILD}" \
+            --exclusions "${JFROG_DL_EXCLUSIONS}" \
+            --flat \
+            "${JFROG_DL_REMOTE_REPO}/"
+        else
+          jfrog rt download \
+            --fail-no-op \
+            --build "${JFROG_DL_BUILD}" \
+            --exclusions "${JFROG_DL_EXCLUSIONS}" \
+            "${JFROG_DL_REMOTE_REPO}/"
+        fi
```

Comment:
Rewrite as a unique command call with a `jfrog_options` valuing either empty or `--flat`.

Reply:
good Idea, I integrated the change here 71253f05797ac3c68dc75d32ef0aa008cebd71b7